CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#5655](https://github.com/thanos-io/thanos/pull/5655) Receive: Fix recreating already pruned tenants.
 
 ### Added
+- [#5653](https://github.com/thanos-io/thanos/pull/5653) Receive: Allow setting hashing algorithm per tenant in hashrings config
```

Comment:
Nit: Can we specify that the hashring config from file has precedent over the flag?

Reply:
Where can I edit the description of the flag?

Reply:
In here: https://github.com/thanos-io/thanos/blob/main/cmd/thanos/receive.go#L820